### PR TITLE
test: Do not retry tests where only css was affected

### PR DESCRIPTION
**Pull request by marusak**

### CODE REVIEW

File: test/verify/check-testlib [hunk 1]
```diff
@@ -167,20 +167,22 @@ class TestRunTest(MachineCase):
         # create affected test for shell changes
         with open(__file__) as f:
             original_test = f.read()
-        affected_testfile = os.path.join(VERIFY_DIR, "test-shell-testlib")
-        writeFile(affected_testfile, original_test.replace("class NoTest", "class Test"))
+        affected_testfile = os.path.join(VERIFY_DIR, "check-shell-testlib")
+        new_content = original_test.replace("class Test", "class NeinTest")
+        writeFile(affected_testfile, new_content.replace("class NeinTestRetryExample", "class TestShell"))
+        os.chmod(affected_testfile, 0o555)
```
Comment (martinpitt) [inline test/verify/check-testlib:173]: Why does it do that? To prevent recursive editing?

**Review by martinpitt — DISMISSED** [review]
Thanks! Well done fixing "teslib" [sic] (no reason re-push just for the typo).

File: test/verify/check-testlib (hunk 1 above)
Comment (marusak) [reply test/verify/check-testlib:173]: To add it +x

**Comment by marusak** [comment]
No test was triggered, not even packit. Seems like github issue. Repushing to trigger it

**Review by martinpitt — APPROVED** [review]

**Pull request merged**